.gitmodules

```diff
@@ -23,7 +23,7 @@
 [submodule "third_party/cares/cares"]
 	path = third_party/cares/cares
 	url = https://github.com/c-ares/c-ares.git
-	branch = cares-1_12_0
+	branch = cares-1_17_1
```

Comment:
@apolcyn should this be `cares-1_17_2` instead?

Reply:
you're right, thanks. Just fixed that

Reply:
Say hi to my first (tiny-little) contribution to gRPC then 🕺 [#28780](https://github.com/grpc/grpc/pull/28780)